extensions/amp-lightbox-viewer/0.1/test/test-amp-lightbox-viewer.js

```diff
@@ -63,8 +80,14 @@ describes.realWin('amp-lightbox-viewer', {
   });
 
   function runTests(autoLightbox) {
```

Comment:
This test is functionally changed because now lightbox builds on `open` and not on the build callback.